src/content/protocols/uniswap-v3.md

```diff
@@ -0,0 +1,136 @@
+---
+protocol: "Uniswap-V3"
+website: "https://blog.uniswap.org/uniswap-v3"
+x: "https://x.com/uniswap"
+github:
+  [
+    "https://github.com/Uniswap/v3-core",
+    "https://github.com/Uniswap/v3-periphery/",
+  ]
+defillama_slug: "uniswap-v3"
+chain: "Ethereum"
+stage: 2
+risks: ["L", "L", "L", "L", "L"]
+author: ["mmilien"]
+submission_date: "2024-11-12"
+publish_date: "2024-12-16"
+acknowledge_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Uniswap v3 is an AMM that builds upon Uniswap v2 by introducing a concentrated liquidity model, providing liquidity providers with granular control over capital allocation. Unlike v2, where liquidity is distributed uniformly across all price ranges, v3 allows LPs to specify custom price ranges in which their liquidity is active. This approach significantly improves capital efficiency, as LPs can concentrate their assets in high-demand price ranges, earning fees only within those specified ranges.
+Uniswap v3 also introduces multiple fee tiers (0.01%, 0.05%, 0.3%, and 1%) to support different asset volatility profiles, allowing LPs to adjust their fee preferences based on expected risk and return. Additionally, it incorporates "range orders," which effectively turn liquidity positions into limit orders, further enhancing LP strategy flexibility.
+The protocol is deployed across multiple chains enabling a wide range of use cases across decentralized finance (DeFi) applications.
+
+# Overview
+
+## Chain
+
+Uniswap is deployed on Ethereum mainnet, Arbitrum, Optimism, Polygon, Base, BNB,
+Avalanche C-Chain, CELO, Blast, ZKsync, Zora, and WorldChain. This analysis is
+based on the contracts deployed on Ethereum mainnet.
+
+> Chain score: L
+
+## Upgradeability
+
+The Uniswap DAO can change parameters such as fees through the GorvernorBravoDelegator contract.
+Apart from the fees set by the governance, the protocol's contracts are immutable. No party is able to pause, revert trade execution, or otherwise change the behavior of the protocol.
+
+No User funds nor unclaimed yield are affected by the upgradability.
+
+Note that a TransparentProxy with the DAO as admin is used for the NonFungibleTokenPositionDescriptor, which is used for token descriptions.
+This should not impact the safety of the protocol.
```

Comment:
"However, this does not impact user funds or otherwise materially change the expected performance of the protocol."